packages/api/src/graphql/invoice.ts

```diff
@@ -68,6 +68,7 @@ export const GraphQLPublicInvoice = new GraphQLObjectType<Invoice, Context>({
     description: {type: GraphQLString},
     paidAt: {type: GraphQLDateTime},
     items: {type: GraphQLNonNull(GraphQLList(GraphQLNonNull(GraphQLInvoiceItem)))},
+    subscriptionID: {type: GraphQLNonNull(GraphQLID)},
```

Comment:
1. Should this be only on public invoice?
2. Shouldn't this be connected with the invoice-item?

Reply:
I understood that if we have the invoice that can have multiple items, actually the items may be connected with different subscriptions.